### DISCUSSION TITLE
Last inserted row id

### DISCUSSION THREAD

**Discussion by mrbu-solutions** [pocketbase/pocketbase#2842] [Q&A]
Hi Pocketbase Team,

Do we have an API for sqlite's sqlite3_last_insert_rowid()?

**Use case:**
Client wants to have an auto-increment field called employee_id with format _"22-000001"_.
How can I do so without querying twice (query last created record to get employee_id, then creating the new record) when inserting new record?
I'm using the JS SDK.

Thank you.

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand your use case and how do you plan to use the `sqlite3_last_insert_rowid()`, but in general we don't provide direct access to the SQLite C APIs (I'm not even sure if it is exposed in the pure Go port [modernc.org/sqlite](https://gitlab.com/cznic/sqlite) that we use by default).

Also keep in mind that the `id` we use differ from the SQLite internal `rowid` and at the moment we don't provide public web api access to that field (_with exception for the log requests APIs for optimization purposes_).
